mediated_transfer.rst

```diff
@@ -0,0 +1,71 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+The protocol supports mediated transfers. Nodes may use them to send payments.
+A :term:`Mediated transfer` may be cancelled and can expire until the initiator reveals the secret.
+
+A mediated transfer is done in two stages, possibly on a series of channels:
```

Comment:
Isn't every mediated transfer done over a series of channels? Can you find a better term than series of channels? Isn't Multi-hop the term used when it comes to Raiden / Lightning?

Reply:
... ok I see in the example below we can also do it with hops ... For a better understanding, can we re-write the example to a mediated transfer with a hop?

Reply:
As I understand it, multi-hop means two or more mediators, so not all mediated transfers are multi-hop.

Reply:
ok I see. Then we should not use the word multi-hop. 

A mediated transfer is a transfer where we need to have at least one hop, right? If there on no hop (e.g. in a direct transfer) we do not need a mediator and therefore, the transfer is not mediated. 
So I would need in the specification, 
- the explanation and definition of what a mediated transfer is 
- a consistent example that reflects this definition

Would you agree to that?

Reply:
My intuition would also have been that a transfer with no mediators is not considered a mediated transfer. However, the spec in its old state stresses that there can be "mediated transfers" with no mediators, and it even gave an example with no mediator (the Alice-Bob example we changed). The point is probably that the mediated transfer protocol with ``LockedTransfer``, ``SecretRequest`` etc messages still works with no mediators (and is currently the way we do direct transfers?)

Reply:
yes that is true. But it is still not a mediated transfer imho. We just use the same protocol for direct transfers as we use it for mediated transfers. But using the same protocol doesn't make it the same think. We shouldn't stress that point tbh, because it makes it only more complicated for the reader without adding value. Thanks for creating the other example!